linux-dotnet-functionapp-on-azure.yml

```diff
@@ -1,6 +1,6 @@
 
 # Action Requires
-# 1. Setup the AZURE_CREDENTIALS secrets in your Github Repository by following README.md
+# 1. Setup the AZURE_CREDENTIALS secrets in your Github Repository
```

Comment:
I would say
1. Setup the AZURE_CREDENTIALS secrets in your GitHub Repository, unless you are using publishing profile; please check the comments in file

Reply:
Another nit picking comment: GitHub not Github :)

Reply:
I will be creating my own PR for these